src/app/blog/[slug]/page.tsx

```diff
@@ -60,15 +61,24 @@ function createBlogPostStructuredData(
 export default function BlogPost({ params }: BlogPostProps) {
   const { slug } = params
   const data = getBlogPostData(slug)
-  const { title, description, image, content } = data
+  const { title, description, image, content, publishedOn, category } = data
 
   return (
     <PageLayout>
       <StructuredDataScript
         structuredData={createBlogPostStructuredData(data)}
       />
-      <PageHeader title={title} description={description} image={image} />
-      {content && <MarkdownContent>{content}</MarkdownContent>}
+      <BreadCrumbs />
```

Comment:
We need to open a ticket to fix the `BreadCrumbs` elsewhere.

Reply:
What do you mean? Where else do we need to include the component?

Reply:
The `Breadcrumbs` are already included in the main layout. This is a bug, remember?. [>>](https://filecoinfoundation.slack.com/archives/C04HCHS4RRS/p1714738648817329?thread_ts=1714479781.531579&cid=C04HCHS4RRS)